todo_app/templates/index.html

```diff
@@ -8,47 +8,66 @@ <h1 class="display-4">To-Do App</h1>
   </div>
  </div>
 	
-    <h2>Create Task</h2>
-    <form action="/create" method="POST">
-        <input type="text" name="title" id="title" value="title">
-        <input type="submit" value="Create">
-    </form>
    
 	<table>
   <tr>
   <div class="row">
     <div class="col-md-12">
-      <th>Items</th>
-	   <th>Status</th>
-	    <th>Item id - Update Status</th>
+     
 	  
 	   </tr>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
 		
-		<ul>
-		{% for item in items %}
-		<li><tr>
-		 <form action="/save" method="POST">
+		<table>
+		<h3><b>To do</h3></b>                                                                                                       
 		
-		<td>{{item.title}}  </td> <td> {{item.status}} </td> 
-		<td><form method="POST">
-        <input name="itemId"value="{{item.id}}">
-        <input type="submit"value="Mark as Complete">
-         </form></td>
-       
+		<tr>{% for listNumber in range(number) %}<tr>   
+<br>		
+	       <tr> {{jsonResponse[listNumber]["name"]}}<tr>
+		 <p>Move from doing to done<p>
+  <form action="/update" method="post">
+        <input type="hidden" name="id" value="{{jsonResponse[listNumber]["id"]}}">
+        <input type="submit" value="Update to done">
     </form>
-		</li>
-		
-		<tr>
+
+   
 </div>
 		{% endfor %}
```

Comment:
Rather than passing through a count, we can iterate over the items in the list directly, so for example we could swap this bit of code for something like the below:
```suggestion
		{% for entry in jsonResponse %} 
			<div>
				<br>		
				<tr> {{entry["name"]}}<tr>
				<p>Move from doing to done<p>
				<form action="/update" method="post">
					<input type="hidden" name="id" value="{{entry['id']}}">
					<input type="submit" value="Update to done">
				</form>
			</div>
		{% endfor %}
```

This would save us having to pass through the number arguments, and simplify this logic a little

Reply:
Thank you for the suggestion, I have updated it with commit